benchmarks/dynamo/common.py

```diff
@@ -4025,6 +4031,18 @@ def run(runner, args, original_dir=None):
         inductor_config.triton.divisible_by_16 = not args.disable_divisible_by_16
         if args.inference:
             inductor_config.freezing = args.freezing
+        if args.inductor_config:
+            for config in args.inductor_config:
+                key, value = config.split("=")
+                typ = type(inductor_config.__getattr__(key))
```

Comment:
Do we want to support config like inductor_config.triton.foo . We need repeatedly call getattr to handle them

Reply:
This already supports that.  My use case is `-c halide.scheduler_cpu=...`.

inductor_config is secretly just a dict, the recursive objects is just syntactic sugar.